dataland-frontend/tests/e2e/utils/LksgUpload.ts

```diff
@@ -84,7 +84,7 @@ function fillSingleProductionSite(): void {
  * This action needs to be recursive as clicking yes on some fields will result in other fields getting visible
  * @param maxCounter the maximum recursion depth before an error is thrown
  */
-function recursivelySelectYesOnAllFields(maxCounter: number): void {
+export function recursivelySelectYesOnAllFields(maxCounter: number): void {
```

Comment:
This method is used by lksg and sfdr now and should therefore be at some other place than this file here.

Reply:
Done.

Reply:
`recursivelySelectYesOnAllFields` still has an `export` before it.

Reply:
Changed it

Reply:
👍